<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="REGIONS TESTCASES"
           tests="4"
           failures="1"
           errors="0"
           skipped="0"
           time="94.813">
  <properties />
            <testcase name="create_a_new_region" time="30.846" classname="layout.Regions_TestCases">
                <failure type="java.lang.AssertionError"
                  message="expected [CODA - Regions] but found [CODA - Regions - Add]"
              >
<![CDATA[java.lang.AssertionError: expected [CODA - Regions] but found [CODA - Regions - Add]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:513)
  org.testng.Assert.assertEqualsImpl(Assert.java:135)
  org.testng.Assert.assertEquals(Assert.java:116)
  org.testng.Assert.assertEquals(Assert.java:190)
  org.testng.Assert.assertEquals(Assert.java:200)
  layout.RegionsAddPage.clickSubmitButton(RegionsAddPage.java:42)
  layout.Regions_TestCases.create_a_new_region(Regions_TestCases.java:20)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:756)
  org.testng.TestRunner.run(TestRunner.java:610)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="database_delete_region" time="2.903" classname="layout.Regions_TestCases" />
                <testcase name="create_a_new_region_without_a_name" time="28.820" classname="layout.Regions_TestCases" />
                <testcase name="create_a_new_region_with_a_duplicate_name" time="32.244" classname="layout.Regions_TestCases" />
      
</testsuite>
